tests/src/Simple/HelloWasm/Program.cs

```diff
@@ -11,7 +11,7 @@ internal static class Program
     private static unsafe void Main(string[] args)
     {
         Add(1, 2);
-      
+        var tempObj = new Object();
```

Comment:
Can you instantiate a class and test a little bit of functionality with it? It might be good to at least cover constructor arguments and reading/writing class instance fields.